sale_stock_restocking_fee_invoicing/models/sale_order.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import UserError
+from odoo.fields import Command
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    def _get_restocking_fee_line_name(self, stock_move):
+        """Return the SO line description for the restocking fee. This method should
+        be called with the customer lang into the context to get the description into
+        the right language
+        """
+        lang = self.partner_id.lang
+        return _(
+            f"Restocking fee for {stock_move.product_uom_qty} "
```

Comment:
You cannot use format like that as values will be reformatted before applying translation.

Either, use `_(%(<variable>)s) % dict(variable=variable) `or` _({variable}).format(variable=variable)`